CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ All notable changes to the Zowe Client Python SDK will be documented in this fil
 
 - Fixed Core SDK package referencing a non-existent version of Secrets SDK.
 
+### Enhancements
```

Comment:
Added changelog also :smile:

Reply:
Sorry for not being explicit, but the changelog has to be towards the top of the file.
and you must include the header `## Recent Changes` 😓 

my apologies for that 😢